views/index.ejs

```diff
@@ -8,10 +8,12 @@
     <h1>Shopping for Cookies</h1>
 
 
-    <strong>Items in cart: <!-- Print the number of items in the cart here --></strong>
+    <strong>Items in cart: <%= cart.length %></strong>
     <ul>
       <!-- List the names of the items in your cart here -->
-
+      <% products.forEach(function(product) { %>
```

Comment:
I don't think you want to loop through `products` here - that's just going to give you a list of all the possible products. Instead, you want to loop through the items in your `cart`. The purpose of line 47 in `app.js` is to create a `cart` variable that's available in the view that is equal to your shopping cart. So you can use that variable and loop over the items in the cart and print those out, rather than printing out the existing items.